### ISSUE TITLE
Should we move to the BSD license?

### ISSUE THREAD

**Issue by FlipperPA** [zostera/django-bootstrap4#1]
With the release, should we switch to the BSD 3-clause license to match what Django uses?

**Comment by dyve** [comment]
Matching Django would be a good idea. In layman's terms, what would change for the user? Link would be fine too, no lengthy explanation necessary.

**Comment by FlipperPA** [comment]
I don't think much of anything would change for the end-user. They are similar licenses, I was just advised that matching the license would simplify things for any company that has to review OS licenses before using packages.

**Comment by dyve** [comment]
Loud and clear. I just started a ticket to switch django-bootstrap3 over to BSD-3. Let's go for BSD-3 here also. Thanks for info. A co-worker of mine just open sourced another Zostera project and used BSD-3 for the same reasons.

**Comment by dyve** [comment]
Closing,

**Comment by dyve** [comment]
Is the LICENSE copied from Django? Because now Django is specifically mentioned in the license text. Is that intentional?

**Comment by FlipperPA** [comment]
Wups! I did grab it directly from Django since I had a copy checked out for another project. I'll review and amend the PR.
